bin/pps_l1c_collector.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2019 - 2021 Pytroll
+
+# Author(s):
+
+#   Erik Johansson <Firstname.Lastname at smhi.se>
+#   Adam Dybbroe <Firstname.Lastname at smhi.se>
+
```

Comment:
```suggestion
```

We don't list the authors names in the header anymore I believe, and instead new authors should be added to the ? AUTHORS.md` file.